basictest.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+actual=$(./less --version | head -1)
+expected='less 6.* (POSIX regular expressions)'
```

Comment:
This is going to break when v700 is released.  Can it be more generic?

Reply:
fixed